### DISCUSSION TITLE
Proposition

### DISCUSSION THREAD

**Discussion by jejemorg** [soundshed/soundshed-app#5] [Ideas]
First great job, it's really comfortable to use on a PC.
From what I understand your idea is to be able to use this amp and feature even if positive grid will stop the service and it's a goo idea.
I think having seen an option for exporting preset but can't find, It would be nice if exist.
Thanks for your work.

**Comment by webprofusion-chrisc** [comment]
Sure thing, your favourites are actually stored in your browser local storage,

 Did you imagine being able to save individual presets to a file or export a bunch of them? Is the aim to preserve them or convert them for something else?

**Reply by jejemorg** [reply]
Yes that is the idea, to be able to locally save the preset into a file and import later.

**Comment by nwfungi** [comment]
Would it be possible to have the search query started with the enter key (pc). If you do a search now and hit enter it takes you back a step. Clicking with mouse does work great, but I am so in the habit of being able to just hit enter in a search box to have it "launch"  Great work BTW

**Reply by lavabyrd** [reply]
#20 has that sorted - really nice suggestion

**Reply by nwfungi** [reply]
Just tried .22, looks like that did the trick
